ENCO2_B_D:
compile_err(boundary_func_11)
python3 ${TEST_BIN}/check.py --check=num  --n=1 --str="boundary_func_11.c:45 error: the pointer >= the upper bounds when accessing the memory and inlined to main"  --result=compile.log
